src/test/java/uk/co/ramp/covid/simulation/population/PensionerTest.java

```diff
@@ -0,0 +1,5 @@
+package uk.co.ramp.covid.simulation.population;
+
+public class PensionerTest {
+
```

Comment:
We possibly want to test the reporting functions here as we did for infants? (although this might be part of the DailyStats testing issue?)

Reply:
Added test for pensioners